src/components/ModalBox/ModalBox.js

```diff
@@ -41,17 +44,23 @@ const Box = styled.View`
   width: 100%;
   margin: auto 0;
   align-self: center;
-  border-radius: 15px;
-  ${({ theme }) => `
-      background-color: ${theme.colors.card};`}
+  border-radius: 10px;
+  background-color: ${themedColors.card};
 `;
 
 const ModalBox = (props: Props) => (
-  <Wrapper>
-    <Box>
-      {props.children}
-    </Box>
-  </Wrapper>
+  <Modal
+    isVisible={props.isVisible}
+    hasBackdrop
+    backdropOpacity={1}
```

Comment:
Can it be unified with other modals on the app? (Set to 0.7?)

Reply:
We want a complete opaque background here.

Reply:
Now I'm not really sure, changed to `0.7` since it makes more sense, and asked Dmitry to confirm. We can update it whenever we have the confirmation.